### DISCUSSION TITLE
running valet share APP_URL doesn't work

### DISCUSSION THREAD

**Discussion by fredbouin** [statamic/cms#6992] [Troubleshooting] [answered]
Hi I'm wondering if there is a typical ENV setup for using valet share.

Right now if I run valet share nothing works because the URL in my ENV file is my local dev environment url, mysite.test
It works once I switch out the APP_URL for the url generated by Ngrok. That's fine, I can do that, but I'm curious if there is a way to setup so I don't have to change it every time I run valet share.

**Comment by edalzell** [comment, marked as answer]
You could try setting the `url` in your `sites.php` to `/`?

**Reply by fredbouin** [reply]
ah wow this is an easy fix. I thought I needed my full url to get a proper path.

**Reply by fredbouin** [reply]
just out of curiosity would it be a proper setup to always leave it like this even in production?

**Reply by jasonvarga** [reply]
It's fine to use `/` most of the time, but it's a good idea to use the full URL.

https://statamic.dev/multi-site#url

If you use an `.env` url you could set it to `/` locally and then the proper url on production.
